.github/workflows/continous-integration.yml

```diff
@@ -50,6 +50,7 @@ jobs:
       backend: ${{ steps.changed-files.outputs.backend || steps.run-all.outputs.backend }}
       docker: ${{ steps.changed-files.outputs.docker || steps.run-all.outputs.docker }}
       docs: ${{ steps.changed-files.outputs.docs || steps.run-all.outputs.docs }}
+      is_pre_release_version: ${{ steps.rasa_check_version_type.outputs.is_pre_release_version }}
```

Comment:
why put it in this action if it's only needed below in the changelog check?

Reply:
I was pre-planning for the future if we need to skip other things for alphas releases :)